lib/craft.rb

```diff
@@ -0,0 +1,8 @@
+class Craft
+  attr_reader :name, :supplies_required
+
+  def initialize(name, supplies_required)
+    @name = name
+    @supplies_required = {:yarn=>20, :scissors=>1, :knitting_needles=>2}
+  end
```

Comment:
With the way this code is written _every_ Craft would have the same supplies of yarn, scissors, and knitting_needles. Instead you want to use the data passed into the initialize argument so that every Craft can be created with different supplies.